plugin/src/sidebar.ts

```diff
@@ -44,6 +44,7 @@ const defaultProject: Project = {
         tile_type: "url",
       },
     ],
+    atmosphere: { shadows: true },
```

Comment:
There is another defaultProjects that you'll need to update too. Please search for it and update there too (I think its in the sidebar's main hooks file)

Reply:
[fix](https://github.com/eukarya-inc/reearth-plateauview/pull/152/commits/439ddec873690071a5f7fb0cc59d7d0abe7044ec)